src/SFA.DAS.Employer.PR.Web/Models/Session/AddTrainingProvidersSessionModel.cs

```diff
@@ -4,10 +4,15 @@ namespace SFA.DAS.Employer.PR.Web.Models.Session;
 
 public class AddTrainingProvidersSessionModel
 {
-    public long? LegalEntityId { get; set; }
-    public string? LegalName { get; set; }
+    public string? EmployerAccountId { get; set; } = null!;
```

Comment:
This is nullable and you have initialised with null negation which is not needed

Reply:
yes

Reply:
making it non-nullable